dotcom-rendering/src/components/LatestLinks.importable.tsx

```diff
@@ -3,23 +3,32 @@ import {
 	space,
 	textSans14,
 	textSansBold14,
+	textSansBold17,
 } from '@guardian/source/foundations';
+import { DottedLines } from '@guardian/source-development-kitchen/react-components';
 import { revealStyles } from '../lib/revealStyles';
 import { useApi } from '../lib/useApi';
 import { palette as themePalette } from '../palette';
 import type { DCRContainerPalette } from '../types/front';
 import { WithLink } from './CardHeadline';
 import { ContainerOverrides } from './ContainerOverrides';
 import { DateTime } from './DateTime';
+import type { Alignment } from './SupportingContent';
 
 type Props = {
 	id: string;
-	direction: 'horizontal' | 'vertical';
+	direction: Alignment;
 	absoluteServerTimes: boolean;
-	isDynamo?: true;
+	isDynamo?: boolean;
 	containerPalette?: DCRContainerPalette;
+	displayHeader?: boolean;
 };
 
+const header = css`
+	padding: 8px 0;
```

Comment:
Should this use a value from Source?

```suggestion
	padding: ${space[2]}px 0;
```